sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/AutocompleteOptions.java

```diff
@@ -1,34 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
```

Comment:
Was this comment manually added?

Reply:
Nope, I've transitioned this class over to being code generated with customizations